nixos/modules/virtualisation/google-compute-image.nix

```
@@ -56,6 +56,33 @@ in
         GZIP compression level of the resulting disk image (1-9).
       '';
     };
+
+    virtualisation.googleComputeImage.contents = mkOption {
+      type = with types; listOf attrs;
+      default = [ ];
+      description = ''
+        The files and directories to be placed in the image.
+        This is a list of attribute sets {source, target, mode, user, group} where
+        `source' is the file system object (regular file or directory) to be
+        grafted in the file system at path `target', `mode' is a string containing
+        the permissions that will be set (ex. "755"), `user' and `group' are the
+        user and group name that will be set as owner of the files.
+        `mode', `user', and `group' are optional.
+        When setting one of `user' or `group', the other needs to be set too.
+      '';
+      example = literalExpression ''
+        [
+          {
+            source = "${./default.nix}";
```

```suggestion
            source = ./default.nix;
```

Is there a reason I am missing that we stringify here or can we keep it simpler?

---

Causes this error:
```
rsync: [sender] link_stat "/build/default.nix" failed: No such file or directory (2)
rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1338) [sender=3.4.1]
error: builder for '/nix/store/bqf80x1hx6rc4wxvqgqw497yq1yg8fl8-google-compute-image.drv' failed with exit code 23;
       last 2 log lines:
       > rsync: [sender] link_stat "/build/default.nix" failed: No such file or directory (2)
       > rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1338) [sender=3.4.1]
```